packages/api/package.json

```diff
@@ -21,7 +21,7 @@
     "clean": "rm -rf dist @types"
   },
   "dependencies": {
-    "better-sqlite3": "^9.3.0",
+    "better-sqlite3": "^9.6.0",
```

Comment:
node-abi was complaining - updating this brought them in line without having to set a override resolve

Reply:
Thanks! Would you mind updating it in `actual-server` too so the two versions align?